frappe/integrations/doctype/webhook/webhook.py

```diff
@@ -112,16 +120,21 @@ def get_context(doc):
 
 
 def enqueue_webhook(doc, webhook) -> None:
-	webhook: Webhook = frappe.get_doc("Webhook", webhook.get("name"))
-	headers = get_webhook_headers(doc, webhook)
-	data = get_webhook_data(doc, webhook)
+	try:
+		webhook: Webhook = frappe.get_doc("Webhook", webhook.get("name"))
+		headers = get_webhook_headers(doc, webhook)
+		data = get_webhook_data(doc, webhook)
+
+		if webhook.is_dynamic_url:
+			request_url = frappe.render_template(webhook.request_url, get_context(doc))
+		else:
+			request_url = webhook.request_url
 
-	if webhook.is_dynamic_url:
-		request_url = frappe.render_template(webhook.request_url, get_context(doc))
-	else:
-		request_url = webhook.request_url
+		r = None
+	except Exception as e:
+		frappe.logger().debug({"enqueue_webhook_error": e})
+		log_request(webhook.name, doc.name, request_url, headers, data, r)
 
-	r = None
```

Comment:
This was added to avoid losing logs. 


If `request.request` doesn't return anything then `r` will be unset and hence exception log wont be created.